content/en/docs/concepts/policy/pod-security-policy.md

```diff
@@ -413,19 +413,27 @@ minimum value of the first range as the default. Validates against all ranges.
 to be used by hostPath volumes. An empty list means there is no restriction on
 host paths used. This is defined as a list of objects with a single `pathPrefix`
 field, which allows hostPath volumes to mount a path that begins with an
-allowed prefix. For example:
+allowed prefix, and a `readOnly` field indicating it must be mounted read-only.
+For example:
 
 ```yaml
 allowedHostPaths:
   # This allows "/foo", "/foo/", "/foo/bar" etc., but
   # disallows "/fool", "/etc/foo" etc.
   # "/foo/../" is never valid.
   - pathPrefix: "/foo"
+    readOnly: true # only allow read-only mounts
 ```
 
-_Note: There are many ways a container with unrestricted access to the host
+{{< warning >}}**Warning:** There are many ways a container with unrestricted access to the host 
```

Comment:
This isn't a complete sentence anymore. :(

Reply:
Really? The text didn't change

Reply:
Ahh you are right, I was just having trouble following the diff.